scheduler-server/sql/scheduler/tables/scheduling_request.sql

```diff
@@ -3,6 +3,9 @@ create type status_t as enum('pending', 'incomplete', 'failed', 'success');
 create table scheduling_request (
   specification_id integer not null,
   analysis_id integer not null,
+  requested_scheduler text not null default 'aerie',
```

Comment:
🔻 I don't think we've fleshed out the design for this feature enough to the point that I would feel comfortable committing to this implementation. Let's hold off on adding the `requested_scheduler` column.

Reply:
To confirm, are we keeping the removal of `scheduling_analysis`, and are intending to revisit the topic of how to represent scheduling requests to external schedulers at some point in the future?

Reply:
Done